bytes/bytes/rabbitmq.py

```diff
@@ -67,6 +47,15 @@ def publish(self, event: Event) -> None:
         pass
 
 
+@lru_cache(maxsize=1)
+def get_connection(queue_uri: str) -> pika.BlockingConnection:
+    logger.info("Connecting to RabbitMQ")
+    connection = pika.BlockingConnection(pika.URLParameters(queue_uri))
+    logger.info("Connected to RabbitMQ")
+
+    return connection
```

Comment:
Since this connection resource is only used/ referenced by `RabbitMQEventManager`, why not just make it an attribute or property of that class? I don't directly see the need for an external cached function for this. Even if it had to be cached, it would still make more sense that it would be a member of `RabbitMQEventManager`

Reply:
Perhaps something like the sample I wrote in https://github.com/minvws/nl-kat-coordination/issues/1162#issuecomment-1590712394 could be used?